ImperatorToCK3/Data_Files/blankMod/output/events/welcome.txt

```diff
@@ -0,0 +1,17 @@
+namespace = welcome
+welcome.1 = {
+	hidden = no
+	type = letter_event
+	opening = { desc = welcome_title }
+	desc = welcome_desc
+	
+	trigger = { }
+	
+	sender = character:converter_team
+	
+	immediate = {
+		character:converter_team = { death = natural }
+	}
+
+	option = { name = welcome_ok }
+}
```

Comment:
Can't we just use the `character_event` type? The converter_team character won't be needed.

Reply:
It's more of a "letter" to the player; if we should change it to a character event, I'll change it

Reply:
Alright.